### ISSUE TITLE
[A11y] Desktop/Mobile Search Results - List: Image Thumbnails

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1831]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Search Results List: https://staging.oregondigital.org/catalog?utf8=%E2%9C%93&search_field=all_fields&q%5B%5D=test

The image thumbnails have the following error:

Failed Instance:
image-alt: Ensures <img> have alternate text or a role of  none or presentation

Example for Image 1
Path:
#document_9z902z939 > .search-result-wrapper.row > .col-sm-4.col-md-3.col-xs-12 > .list-thumbnail > a > img

Snippet:
<img src="/downloads/h989r329k?file=thumbnail">

### Suggested Fix

Fix ONE of the following:

- Element does not have an alt attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by CGillen** [comment]
Fixed by #2037
